develop/tutorials/articles/192-item-selector/01-selecting-entities-using-the-item-selector-api.markdown

```diff
@@ -0,0 +1,356 @@
+    
+2.  Add the following AUI taglib directive to the top of your JSP:
+
+        <%@ taglib prefix="aui" uri="http://liferay.com/tld/aui" %>
+        
+3.  Add the AUI script for your Item Selector dialog trigger.
+
+    For example, you can use the following script to add a button:
+    
+            <aui:button name="name" value="value" id="triggerID" />
+            
+4.  Use the following pattern to attach a click event to the trigger you just 
+    created, to open the Item Selector dialog.
+
+        <aui:script use="liferay-item-selector-dialog">
+        
+        $('#<portlet:namespace />triggerID').on(
+        
+          'click',
+        
+          function(event) {
+        
+             var itemSelectorDialog = new A.LiferayItemSelectorDialog(
+        
+                {
+        
+                   eventName: 'itemSelectedEventName',
+        
+                   on: {
+        
+                      selectedItemChange: function(event) {
+        
+                         var selectedItem = event.newVal;
+        
+        var itemValue = JSON.parse(selectedItem.value);
+        
+        itemSrc =  itemValue.url;
+        
+        [use itemSrc as needed]
+        
+                         }
+        
+                      }
+        
+                   },
+        
+                   title: '<liferay-ui:message key="language-message-key" />',
+        
+                   url: '<%= itemSelectorURL.toString() %>'
+        
+                }
+        
+             );
+        
+             itemSelectorDialog.open();
+        
+          }
+        
+        );
+        
+        </aui:script>
+    
+    For example, using the Java code example in the last section, the logic 
+    would be as follows:
+    
+        <aui:button name="chooseImage" value="Choose" />
+    
+        <aui:script use="liferay-item-selector-dialog">
+        
+        $('#<portlet:namespace />chooseImage').on(
+        
+          'click',
+        
+          function(event) {
+        
+             var itemSelectorDialog = new A.LiferayItemSelectorDialog(
+        
+                {
+        
+                   eventName: 'sampleTestSelectItem',
+        
+                   on: {
+        
+                      selectedItemChange: function(event) {
+        
+                         var selectedItem = event.newVal;
+        
+        var itemValue = JSON.parse(selectedItem.value);
+        
+        itemSrc =  itemValue.url;
+        
+        [use itemSrc as needed]
+        
+                         }
+        
+                      }
+        
+                   },
+        
+                   title: '<liferay-ui:message key="select-image" />',
+        
+                   url: '<%= itemSelectorURL.toString() %>'
+        
+                }
+        
+             );
+        
+             itemSelectorDialog.open();
+        
+          }
+        
+        );
+        
+        </aui:script>
```

Comment:
I think that one of the cases would be to use the itemSrc to update an existing <img /> tag. Other option could be to use an alert to prompt the url in a dialog. 
I don't know what kind of example we could include here because the important thing to highlight is that we have the value of itemSrc.